packages/core/src/lib/Select/SelectBeta/components/SelectMenuList.tsx

```diff
@@ -68,7 +68,7 @@ const SelectMenuList: React.FunctionComponent<ISelectExtendedProps> = ({
         if (wrapper) {
             const el = wrapper.querySelector(selector);
             if (el) {
-                el.scrollIntoView(false);
+                el?.scrollIntoView?.(false);
```

Comment:
i didnt know `scrollIntoView` was a function now, we used to do this manually. It seems well supported but there is some notes about Safari

https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView

Reply:
the function is supported only the options parameter is not, just to be on the safer side I added optional chaining too.
I tested it on safari and it is working 👍🏻